README.md

```diff
@@ -1 +1,81 @@
+  - If the above parsing failed for any reason, then if the API responded with a 500 Status code, your method will throw an InternalServerError, otherwise, an ErrorPayloadParsingException with the status code of the response will be thrown.
+- You can receive two connectivity related errors, one when the device is not connected to the network and one when a timeout happens.
+- If any other error/exception occurs during the HTTP call, your method will throw a ServerCommunicationException, wrapping the original exception. 
+
+## Configuration
+
+  - **I want to handle serialization related exceptions.**
+    - **error-handler** is library independent, and by default any unknown exception will be wrapped into a **ServerCommunicationException**. If you want to handle your own serialization exception, ex. A JsonDataException with Moshi, then you have to provide a  **SerializationExceptionConverter** implementation which converts your exact serialization exception to a generic one which will be consumed by the call adapter. You can do all this in 1 step, in the builder, like this:
+      
+      ```kotlin
+      RestHandlerCallAdapter.Builder()
+          .addSerializationConverter(object : SerializationExceptionConverter {
+              override fun convert(networkException: NetworkException): NetworkException {
+                  // Do your conversion here!
+                  return YourCustomNetworkException
+              }
+          })
+      ```
+    
+      
+  - **I have a business requirement to log errors which occur with the network call.**
+    - In that case, you just have to provide a NetworkErrorLogger implementation to the RestCallAdapter.Builder . There's  no limit on how many NetworkErrorLoggers you can have.
+  - **I don’t like working with Any for the error body in the ApiError, I want a model class.**
+    - Don’t worry, we have you covered! In this case you have to provide a ParsedApiErrorConverter implementation where you receive that Any and you can convert to your own exception with any model you want.
+  - **I have a custom exception instead of ApiError, but I want to log it.**
+    - Nothing is easier than that. Just make sure that your custom exception implements the Loggable interface, and the rest is handled during the error conversion. Every RemoteException which implements the Loggable interface is guaranteed to be logged.
+  - **I don’t want to handle all of the possible exceptions with `try {} catch() {}` outside the data layer.**
+    - In this case, you can use the ResultWrapper from the error-handler-core if you don’t already have such a mechanism in your project, together with the wrapToResult helper function.
```

Comment:
I would change the order in this sentence like this: 
_In this case, you can use the ResultWrapper from the error-handler-core together with the wrapToResult helper function, if you don’t already have such a mechanism in your project._